test/helpers/kubectl.go

```diff
@@ -940,36 +940,30 @@ func (kub *Kubectl) CiliumPolicyAction(namespace, filepath string, action Resour
 
 // CiliumReport report the cilium pod to the log and appends the logs for the
 // given commands.
-func (kub *Kubectl) CiliumReport(namespace string, commands ...[]string) {
+func (kub *Kubectl) CiliumReport(namespace string, commands ...string) {
 	pods, err := kub.GetCiliumPods(namespace)
 	if err != nil {
 		kub.logger.WithError(err).Error("cannot retrieve cilium pods on ReportDump")
 		return
 	}
 
 	wr := kub.logger.Logger.Out
-	fmt.Println("===================== TEST FAILED =====================")
-	fmt.Fprint(wr, "===================== TEST FAILED =====================\n")
+	fmt.Fprintln(wr, "===================== TEST FAILED =====================")
 
-	data := kub.Exec(fmt.Sprintf("%s get pods -o wide", KubectlCmd))
-	fmt.Fprintln(wr, data.Output())
+	// Dump a human readable view of pods in the test-output log
+	_ = kub.Exec(fmt.Sprintf("%s get pods -o wide --all-namespaces", KubectlCmd))
 
 	for _, pod := range pods {
-		res := kub.Logs(namespace, pod)
-		fmt.Fprintln(wr, res.Output())
-
 		for _, cmd := range commands {
 			command := fmt.Sprintf("%s exec -n %s %s -- %s", KubectlCmd, namespace, pod, cmd)
-			out := kub.Exec(command)
-			fmt.Fprintln(wr, out.CombineOutput())
+			_ = kub.Exec(command)
 		}
 	}
+
 	kub.DumpCiliumCommandOutput(namespace)
 	kub.GatherLogs()
 	kub.CheckLogsForDeadlock()
-	fmt.Fprint(wr, "===================== EXITING REPORT GENERATION =====================")
-	fmt.Println("===================== EXITING REPORT GENERATION =====================")
-
+	fmt.Fprintln(wr, "===================== EXITING REPORT GENERATION =====================")
```

Comment:
Please revert removal of `fmt.Println("===================== EXITING REPORT GENERATION =====================")` . It makes it easy to see where tests fail on Jenkins, as it is not immediately obvious.

Reply:
Umm, the print is in there to the GinkgoWriter. If you don't use the GinkgoWriter the os.Stdout is not in the same order, and some functions are not in the correct place. 

All GinkgoWriter is dump to stdout in the correct order.